packages/usuario/src/drivers/db/models/index.js

```diff
@@ -0,0 +1,7 @@
+const { Usuario, UsuarioSchema } = require('./usuario');
+
+function setupModels(sequelize) {
+  Usuario.init(UsuarioSchema, Usuario.config(sequelize));
+}
+
+module.exports = setupModels;
```

Comment:
Could we add a new line here? You can read why it is needed [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)